articles/tutorials/qft-tutorial.md

```diff
@@ -466,22 +598,24 @@ The final operation code should look like:
             DumpMachine();
 
             for(i in IndexRange(qs)) {
-                set r w/= i <- M(qs[i]);
+                set resultArray w/= i <- M(qs[i]);
             }
 
             Message("After measurement: ");
             DumpMachine();
 
             ResetAll(qs);
         }
-        return r;
+        Message("Post-QFT measurement results [qubit0, qubit1, qubit2]: ");
```

Comment:
This doesn't print out any results, did you mean to use an interpolated string?
```suggestion
        Message($"Post-QFT measurement results [qubit0, qubit1, qubit2]: {resultArray}");
```

Reply:
I think lines were miss-placed while I was editing: first, print the message and then return `r` (that should be `resultArray`) that prints automatically the array of results.